nifpga/session.py

```diff
@@ -390,8 +413,250 @@ def read(self):
             (list): The data in the register in a python list.
         """
         buf = self._ctype_type()
-        self._read_func(self._session, self._resource, buf, len(self))
-        return [bool(elem) if self._datatype is DataType.Bool else elem for elem in buf]
+        read_func = self._get_read_function()
+        read_func(self._session, self._resource, buf, len(self))
+        val = [bool(elem) if self._datatype is DataType.Bool else elem for elem in buf]
+        return val
+
+
+class _FxpRegister(_Register):
+    """
+    _FxpRegister is a private class that does all the work of converting the
+    LabVIEW fixed point data type into something more native to python. As a
+    user you will read and write to this register just as you would any other
+    register. Given the nature of fixed point there are a few caveats.
+
+    Just like the other registers, we do not support users creating their
+    instances of _FxpRegister, but after opening a session to a valid bitfile
+    the session.registers property will contain all registers fixed point
+    included.
+
+    Fixed point registers should be easily used from python with a few caveats:
+        1. Trying to write a value that does not conform to boundaries of the
+        defined register, will be coerced just as it would in labVIEW. Input
+        a value that needs to be coerced will result in a warning to the user.
+        A value is to be coerced if it is not a multiple of the delta value, or
+        if it exceeds the minimum or maximum values.
+        2. When writing to fixed point registers with the overflow status
+        enabled, you must pass in the tuple (overflow bool, value). Failing to
+        pass in the overflow boolean will raise an exception.
```

Comment:
For #2, If its easy, to make it a little more user friendly I would say default the "overflow" to False if the user doesn't provide it.  I feel like writing with an overflow is an edge case.

Reply:
Updated to throw a warning instead.